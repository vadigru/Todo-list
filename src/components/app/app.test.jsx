import React from "react";
import renderer from "react-test-renderer";

import App from "./app";

it(`Should render App component`, () => {
  const tree = renderer
    .create(
        <App />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
