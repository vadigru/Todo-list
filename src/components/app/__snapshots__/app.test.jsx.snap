// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render App component 1`] = `
Array [
  <header
    className="header"
  >
    <h1
      className="header__title"
    >
      TODO
    </h1>
    <h2
      className="header__tasks-count"
    >
      You have 
      0
       active task/s
    </h2>
  </header>,
  <main
    className="content"
  >
    <div
      className="container"
    >
      <form
        className="input"
        onSubmit={[Function]}
      >
        <input
          autoComplete="off"
          className="input__text"
          id="input-task"
          onChange={[Function]}
          placeholder="input task"
          type="text"
          value=""
        />
        <label
          htmlFor="input-task"
        />
        <button
          className="input__submit"
          type="submit"
        >
          <img
            alt="add task"
            src="img/icons8-plus.svg"
          />
        </button>
      </form>
      <ul
        className="tasks"
        data-rbd-droppable-context-id="0"
        data-rbd-droppable-id="dp1"
      />
    </div>
  </main>,
]
`;
